src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -341,24 +341,33 @@ export default class SurvivalChart
         return data;
     }
 
+    private get showPValueText() {
+        // 1. p value is not null or undefined
+        // 2. every group has at least 10 cases
+        return (
+            !_.isNil(this.props.pValue) &&
```

Comment:
https://lodash.com/docs/4.17.15#isNil

_.isNil(value): Checks if value is null or undefined.